rust-connectors/sources/http/src/formatter.rs

```diff
@@ -0,0 +1,43 @@
+// Pre-Allocations ::with_capacity()
+#![allow(clippy::vec_init_then_push)]
```

Comment:
I wonder if we change line 12 to `vec![];` we wouldn't need this.

Reply:
:+1: Seems I dared clippy for no reason there.. transferred the two push straight into vec![] - good catch